src/animation/index.js

```diff
@@ -128,17 +101,43 @@ exports = {
 
   /**
    * Animate an actor around an x, y position
-   * @func AnimationHelper.rotateAround
-   * @arg {Object}   opts
-   * @arg {Actor}    opts.target        - The target actor you wish to move
-   * @arg {number}   opts.x             - The x coordinate of the rotation point
-   * @arg {number}   opts.y             - The y coordinate of the rotation point
-   * @arg {number}   [opts.duration]    - The duration of the animation
-   * @arg {number}   [opts.rotation]    - The amount to rotate
-   * @arg {Function} [opts.transition]  - The transition to use for easing
-   * @arg {Function} [opts.rotateActor] - The transition to use for easing
+   * @param {object}   opts
+   * @param {Actor}    opts.target        The target actor you wish to move
+   * @param {number}   opts.x             The x coordinate of the rotation point
+   * @param {number}   opts.y             The y coordinate of the rotation point
+   * @param {number}   [opts.duration]    The duration of the animation
+   * @param {number}   [opts.rotation]    The amount to rotate
+   * @param {function} [opts.transition]  The transition to use for easing
+   * @param {function} [opts.rotateActor] The transition to use for easing
```

Comment:
Leftover paste from before, should be updated with 'Set to true to update actor rotation as well as position', or something like that